getting-started/proc-configuring-keycloak-with-oidc.adoc

```diff
@@ -0,0 +1,83 @@
+[id="proc-configuring-keycloak-with-oidc_{context}"]
+
+= Configuring Keycloak with the OIDC authentication provider
+
+//You can use Keycloak to authenticate with {product-short} using the OIDC authentication provider. 
+
+{product} includes an OIDC authentication provider that can authenticate users by using Keycloak.
+
+
+
+[IMPORTANT]
+====
+The user that you create in Keycloak must also be available in the {product-short} catalog.
+====
+////
+.Prerequistes
+. You configured {product-short} with a custom config map and secret.
+
+.Procedure
+////
+
+.Procedure
+. In Keycloak, create a new realm, for example `RHDH`.
+. Add a new user.
+
+*Username*:: Username for the user, for example: `rhdhuser`
+*Email*:: Email address of the user.
+*First name*:: First name of the user.
+*Last name*:: Last name of the user.
+*Email verified*:: Toggle to *On*.
+
+. Enter the *Password* for the user account and toggle *Temporary* to *Off*.
```

Comment:
Could you add more information for the user like click on 'Create' button, and on the next page, select the 'Credentials' tab. Then, click on 'Set Password'